src/Fractal.php

```diff
@@ -341,14 +365,18 @@ public function createData()
 
         $this->manager->setRecursionLimit($this->recursionLimit);
 
-        if (! is_null($this->includes)) {
+        if ((bool) $this->includes) {
```

Comment:
I'd go with empty as it is much faster, but yeah i agree that casting is not the most elegant solution.